msal/application.py

```diff
@@ -1628,7 +1638,7 @@ def acquire_token_by_username_password(
         """
         claims = _merge_claims_challenge_and_capabilities(
                 self._client_capabilities, claims_challenge)
-        if self._enable_broker:
+        if False:  # Disabled, for now. It was if self._enable_broker:
```

Comment:
FYI. In v1, we also expose broker for InteractiveBrowserCred, there is no support for username password for broker in Python identity too.

Reply:
FYI v2: We [reverted course again](https://github.com/AzureAD/microsoft-authentication-library-for-python/issues/702), back to the original.

CC: @jiasli 

Note: New comments, if any, shall be posted into that new feature request 702.